### PR TITLE
doc update for 0.0.1a release

### PR DESCRIPTION
**Pull request by MartinPerez**
Update all docs for release.

### CODE REVIEW

**Comment by MartinPerez** [comment]
still work in progress, replaced logo by Alpha version warning and modified examples hierarchy.

**Comment by MartinPerez** [comment]
@bthirion I fixed the documentation in several places: version, source references, authors, etc.
I moved plotting functions (design matrix plot) to reporting.py. (opened an issue to add smoke tests for remaining plotting functions)
Implemented examples hierarchy and replaced logo with alpha version warning.
I also added a content table and moved imports in the tutorials.

Only checking/improving the BIDS tutorials explanations is pending.

File: examples/03_second_level_models/README.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Second level analysis examples
+------------------------------
+
+These are examples focused on showcasing Second Level Models functionality and group level analysis.
```
Comment (bthirion) [inline examples/03_second_level_models/README.txt]: 'second level models'

File: examples/02_first_level_models/README.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+First level analysis examples
+------------------------------
+
+These are examples focused on showcasing First Level Models functionality and single subject analysis.
```
Comment (bthirion) [inline examples/02_first_level_models/README.txt]: '[...] first level models [...]'

File: examples/04_low_level_functions/README.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Low level function examples
+----------------------------
+
+These are examples focused on showcasing low level functionality like design matrix creationg, HRF functions and others.
```
Comment (bthirion) [inline examples/04_low_level_functions/README.txt]: 'creation'

**Comment by bthirion** [comment]
Tiny glitches, but looks good otherwise.

**Comment by MartinPerez** [comment]
@bthirion I fixed glitches, created link to contributors page, expanded a bit the tutorials explanations and updated the changelog.

**Comment by MartinPerez** [comment]
by merging this and #137 we are only missing making nistats/nistats.github.io and pushing to pypi (I already updated version in files). @bthirion we merge #137 first and rebase this one?

**Comment by MartinPerez** [comment]
@bthirion rebased and ready for final check.

**Comment by bthirion** [comment]
great. Merging.

**Pull request merged**